From e916a79966a3c139e2e9620ce2fd3afcb3a2d204 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sun, 9 May 2021 17:18:59 -0400
Subject: [PATCH 160/230] Revert "SystemUI: Move network traffic monitor under
 same layout as clock"

This reverts commit 5a28ff769b487f003dc72cbd0ede6e8da4e605b3.
---
 packages/SystemUI/res/layout/status_bar.xml   | 34 +++++++++----------
 .../statusbar/phone/PhoneStatusBarView.java   | 10 +++---
 2 files changed, 22 insertions(+), 22 deletions(-)

diff --git a/packages/SystemUI/res/layout/status_bar.xml b/packages/SystemUI/res/layout/status_bar.xml
index a55dad43b64..22e82ae8df8 100644
--- a/packages/SystemUI/res/layout/status_bar.xml
+++ b/packages/SystemUI/res/layout/status_bar.xml
@@ -109,6 +109,22 @@
             android:clipChildren="false"
             android:gravity="center_horizontal|center_vertical"/>
 
+        <com.android.systemui.statusbar.LineageStatusBarItemHolder
+            android:id="@+id/network_traffic_holder"
+            android:layout_width="wrap_content"
+            android:layout_height="match_parent"
+            android:gravity="start|center_vertical"
+            >
+
+            <org.lineageos.internal.statusbar.NetworkTraffic
+                android:id="@+id/network_traffic"
+                android:layout_width="wrap_content"
+                android:layout_height="match_parent"
+                android:maxLines="2"
+                android:gravity="center_vertical|start" />
+
+        </com.android.systemui.statusbar.LineageStatusBarItemHolder>
+
         <com.android.keyguard.AlphaOptimizedLinearLayout android:id="@+id/system_icon_area"
             android:layout_width="0dp"
             android:layout_height="match_parent"
@@ -122,7 +138,7 @@
     </LinearLayout>
 
     <com.android.keyguard.AlphaOptimizedLinearLayout
-        android:id="@+id/centered_area"
+        android:id="@+id/center_clock_layout"
         android:gravity="center"
         android:orientation="horizontal"
         android:layout_width="match_parent"
@@ -137,22 +153,6 @@
             android:gravity="center"
             android:visibility="gone"
         />
-
-        <com.android.systemui.statusbar.LineageStatusBarItemHolder
-            android:id="@+id/network_traffic_holder"
-            android:layout_width="wrap_content"
-            android:layout_height="match_parent"
-            android:gravity="center"
-            >
-
-            <org.lineageos.internal.statusbar.NetworkTraffic
-                android:id="@+id/network_traffic"
-                android:layout_width="wrap_content"
-                android:layout_height="match_parent"
-                android:maxLines="2"
-                android:gravity="center" />
-
-        </com.android.systemui.statusbar.LineageStatusBarItemHolder>
     </com.android.keyguard.AlphaOptimizedLinearLayout>
 
     <ViewStub
diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/phone/PhoneStatusBarView.java b/packages/SystemUI/src/com/android/systemui/statusbar/phone/PhoneStatusBarView.java
index 3debe9263eb..bb38f652603 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/phone/PhoneStatusBarView.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/phone/PhoneStatusBarView.java
@@ -411,12 +411,12 @@ public class PhoneStatusBarView extends PanelBar implements Callbacks {
 
         setPadding(padding.first, getPaddingTop(), padding.second, getPaddingBottom());
 
-        // Apply negative paddings to centered area layout so that we'll actually be on the center.
+        // Apply negative paddings to center clock layout so that we'll actually be on the center.
         final int winRotation = getDisplay().getRotation();
-        LayoutParams centeredAreaParams =
-                (LayoutParams) findViewById(R.id.centered_area).getLayoutParams();
-        centeredAreaParams.leftMargin = winRotation == Surface.ROTATION_0 ? -padding.first : 0;
-        centeredAreaParams.rightMargin = winRotation == Surface.ROTATION_0 ? -padding.second : 0;
+        LayoutParams centerClockParams =
+                (LayoutParams) findViewById(R.id.center_clock_layout).getLayoutParams();
+        centerClockParams.leftMargin = winRotation == Surface.ROTATION_0 ? -padding.first : 0;
+        centerClockParams.rightMargin = winRotation == Surface.ROTATION_0 ? -padding.second : 0;
     }
 
     public void setHeadsUpVisible(boolean headsUpVisible) {
-- 
2.25.1

